### DISCUSSION TITLE
External Energy Meter Data from Daikin.

### DISCUSSION THREAD

**Discussion by pwwukpw** [raomin/ESPAltherma#186] [Q&A]
Has anyone managed to get an energy meter mesuring total kwh into HP system, that has a data connection to the heat pump, as a sensor in this integration? measurement?![20221111_172306.jpg](https://user-images.githubusercontent.com/88422402/201395328-c1cc6cf1-daba-4d32-92a9-19b0800705ea.jpg)

**Comment by pwwukpw** [comment]
@raomin beer truly purchased. Great work ! 

Very happy to buy another if you had a moment to assist in finding the code to read the external meter wired to the Dakin into your integration for accurate COP measurements. 

Otherwise have to buy another Meyer which would seem a shame as ine external meter already reporting to Dakin motherboard.

**Comment by glosair** [comment]
There are a lot of ESP based power meters that can have new firmware flashed to send MQTT power data to HA, you can then do the maths there to combine the data ESPAltherma?

**Comment by pwwukpw** [comment]
Hi . thank you for your reply.

I already have a meter that is wired into the Monoblock 3 outdoor uinit ..
was hoping to read the values from within the integration.

BW

Peter
